guide/interactions/replying-to-slash-commands.md

```diff
@@ -0,0 +1,202 @@
+# Replying to Slash Commands
+
+Discord provides users the option to create client-integrated slash commands. In this section you will be learning how to respond to these commands using discord.js!
+
+::: tip
+You need to have at least one slash command set-up for your application to follow the instructions on this page.
+:::
+
+## Receiving Interactions
+
+Every slash command is an `interaction`, so to respond to a command you need to setup an event listener that will execute code when your application receives an interaction:
+
+```js
+client.on('interaction', interaction => {
+	console.log(interaction);
+});
+```
+
+::: warning
+Not every `interaction` is a clash command, so you need to make sure you that you know which type of Interaction you are dealing with!
+:::
+
+Let's make sure you only receive slash commands by making use of the `interaction.isCommand()` method:
+
+```js
+client.on('interaction', interaction => {
+    // We return if our interaction is not a slash command
+    if (!interaction.isCommand()) return; 
+	console.log(interaction);
+});
+```
+
+
+## Responding to a Command
+
+There are multiple ways of responding to a slash command, we will be covering each of these in the following segments.
+The most common way of sending a response is by using the `interaction.reply()` method:
+
+::: warning
+Initially an Interaction token is only valid for 3 Seconds, so that's the timeframe in which you are able to use the `interaction.reply()` method. We will be covering responses that require more time in the Deferred Responses section of this page.
```

Comment:
```suggestion
Initially an Interaction token is only valid for three seconds, so that's the timeframe in which you are able to use the `interaction.reply()` method. Responses that require more time ("Deferred Responses") are explained later in this page.
```